snuba/cli/multistorage_consumer.py

```diff
@@ -289,30 +257,91 @@ def handler(signum: int, frame: Any) -> None:
     processor.run()
 
 
-def build_multistorage_streaming_strategy_factory(
+LogicalTopicTuple = Tuple[
+    Topic,
+    Optional[Topic],
+    Optional[Topic],
+    Optional[Callable[[], DeadLetterQueuePolicy]],
+]
+
+
+def get_logical_topics(
```

Comment:
How about calling this `get_consumer_config()`. Not everything here is a topic, and it seems likely we will return more config from here in the future.

Reply:
right, makes sense